stickyWallet/files/Configuration.kt

```diff
@@ -1,151 +1,120 @@
-            "&2&l>> &a/currency color <plural> <color> &8- &7Select a color for a currency.",
-            "&2&l>> &a/currency colorlist &8- &7List of Colors.",
-            "&2&l>> &a/currency decimals <plural> &8- &7Enable decimals for a currency.",
-            "&2&l>> &a/currency payable <plural> &8- &7Set a currency payable or not.",
-            "&2&l>> &a/currency default <plural> &8- &7Set a currency as default.",
-            "&2&l>> &a/currency startbal <plural> <amount> &8- &7Set the starting balance for a currency.",
-            "&2&l>> &a/currency setrate <plural> <amount> &8- &7Sets the currency's exchange rate."));
-
-        config.addDefault(path + "usage.currency_create", "&2&l>> &a/currency create <singular> <plural> &8- &7Create a currency.");
-        config.addDefault(path + "usage.currency_delete", "&2&l>> &a/currency delete <plural> &8- &7Delete a currency.");
-        config.addDefault(path + "usage.currency_convert", "&2&l>> &a/currency convert <method> &8- &7Convert storage method. WARN: Take backups first and make sure the storage you are switching to is empty!");
-        config.addDefault(path + "usage.currency_backend", "&2&l>> &a/currency backend <method> &8- &7Switch backend. This does not convert.");
-        config.addDefault(path + "usage.currency_view", "&2&l>> &a/currency view <plural> &8- &7View information about a currency.");
-        config.addDefault(path + "usage.currency_list", "&2&l>> &a/currency list &8- &7List of currencies.");
-        config.addDefault(path + "usage.currency_symbol", "&2&l>> &a/currency symbol <plural> <char|remove> &8- &7Select a symbol for a currency or remove it.");
-        config.addDefault(path + "usage.currency_color", "&2&l>> &a/currency color <plural> <color> &8- &7Select a color for a currency.");
-        config.addDefault(path + "usage.currency_colorlist", "&2&l>> &a/currency colorlist &8- &7List of Colors.");
-        config.addDefault(path + "usage.currency_payable", "&2&l>> &a/currency payable <plural> &8- &7Set a currency payable or not.");
-        config.addDefault(path + "usage.currency_default", "&2&l>> &a/currency default <plural> &8- &7Set a currency as default.");
-        config.addDefault(path + "usage.currency_decimals", "&2&l>> &a/currency decimals <plural> &8- &7Enable decimals for a currency.");
-        config.addDefault(path + "usage.currency_startbal", "&2&l>> &a/currency startbal <plural> <amount> &8- &7Set the starting balance for a currency.");
-        config.addDefault(path + "usage.currency_setrate", "&2&l>> &a/currency setrate <plural> <amount> &8- &7Sets the currency's exchange rate.");
-
-        config.options().copyDefaults(true)
-        plugin.saveConfig()
-        plugin.reloadConfig()
-    }
-
-    companion object {
-        private const val path = "messages."
-    }
-
-}
\ No newline at end of file
+package stickyWallet.files
```

Comment:
smh what is this doing here

Reply:
Those are there for further reference once I can get off my lazy butt and migrate translations